unit/test-suffix-si-parse.c

```diff
@@ -41,7 +41,7 @@ static struct tonum_test tonum_tests[] = {
 void tonum_test(struct tonum_test *test)
 {
 	__u64 num;
-	bool suffixed;
+	bool suffixed = false;
```

Comment:
Why not fixing the conversion function? I should always set suffixed when it doesn't return an error.

Reply:
Fixed the patch and chaged the test also again as the expected value for "6.14" to `6` from `0` since the `suffix_si_parse` and its `strtod` behavior was changed to return `6` from `0`. I am not sure the behavior change reason but seems not caused by the fix changes.

Reply:
This could be caused by the `suffixed` variable to false. This looks wrong too. Let me address this.

Reply:
Thank you. Noted it.

Reply:
I've re implemented the function. So don't invest more time on it :)

Reply:
Okay I will do close this pull request. Thank you.

Reply:
Thanks anyway. Your patch is a good fix for the v2.3 release :)